posthog/clickhouse/system_status.py

```diff
@@ -49,23 +50,28 @@ def system_status() -> Generator[SystemStatusRow, None, None]:
         "value": get_event_count_month_to_date(),
     }
 
-    recordings_status = get_recording_status_month_to_date()
+    recording_status = None
+    try:
+        recordings_status = get_recording_status_month_to_date()
```

Comment:
here we were getting

```
DB::Exception: Unknown table expression identifier 'session_replay_events' in scope SELECT countDistinct(session_id), sum(event_count), sum(message_count), formatReadableSize(sum(size)) FROM session_replay_events WHERE toStartOfMonth(min_first_timestamp) = toStartOfMonth(now()). 
```

we _shouldn't_ be getting that
but my guess is the migration is failing or not finished by the time the test runs